code/modules/modular_computers/file_system/programs/command/card.dm

```diff
@@ -41,17 +41,14 @@
 	data["mmode"] = mod_mode
 	data["centcom_access"] = is_centcom
 
-	data["command_jobs"] = format_jobs(SSjobs.titles_by_department(COM))
-	data["support_jobs"] = format_jobs(SSjobs.titles_by_department(SPT))
-	data["engineering_jobs"] = format_jobs(SSjobs.titles_by_department(ENG))
-	data["medical_jobs"] = format_jobs(SSjobs.titles_by_department(MED))
-	data["science_jobs"] = format_jobs(SSjobs.titles_by_department(SCI))
-	data["security_jobs"] = format_jobs(SSjobs.titles_by_department(SEC))
-	data["exploration_jobs"] = format_jobs(SSjobs.titles_by_department(EXP))
-	data["service_jobs"] = format_jobs(SSjobs.titles_by_department(SRV))
-	data["supply_jobs"] = format_jobs(SSjobs.titles_by_department(SUP))
-	data["civilian_jobs"] = format_jobs(SSjobs.titles_by_department(CIV))
+	var/list/dept_list = SSdepartments.departments
+	for(var/dept_key in dept_list)
+		var/datum/department/dept = dept_list[dept_key]
+		data["[dept.refrence]_jobs"] = format_jobs(SSjobs.titles_by_department(dept.refrence)) //get all jobs for all deparments
 	data["centcom_jobs"] = format_jobs(get_all_centcom_jobs())
+	data["test"] = {"<tr><th style="color: '#808080';">Enclave</th><td>{{for data.enclave_jobs}}{{if index && index % 6 === 0}}{{/if}}
```

Comment:
Did this sneak in?

Reply:
Oh, that's supposed to be gone, it didn't work. I was seeing if I could handle the issue in a for loop in dm rather than tmpl. I could not.